test/common.js

```diff
@@ -64,18 +66,19 @@ function rmdirSync(p, originalEr) {
 }
 
 exports.refreshTmpDir = function() {
+  if (!fs.existsSync(testRoot)) {
```

Comment:
[`existsSync()` is deprecated](https://nodejs.org/api/fs.html#fs_fs_existssync_path) so maybe use one of the suggested alternatives?